client/app/views/index.js

```diff
@@ -4,43 +4,43 @@ var _ = require('lodash');
 var h = require('../helpers/helpers');
 var renderPipeline = require('./components/pipelineView').render;
 
-var appendError = function appendError(err) {
-  h.$append('#content-container', 'Fetching pipelines has a problem. Please try again');
-  h.trace(err);
+var appendError = function(message) {
+  var errorMsg = message ||  'An error occurred loading the page. Please try again.';
+  return function appendError(err) {
+    h.$append('#content-container', errorMsg);
+    h.trace(err);
+  };
 };
 
 var fetchPipelinesAsync = function fetchPipelinesAsync() {
```

Comment:
fwiw, ES6 standardizes functions' name property which provides a pretty good introspection (once implemented by browsers) as long as that name is taken into consideration in devtools